tests/panels/test_sql.py

```diff
@@ -655,8 +655,8 @@ def test_flat_template_information(self):
         template_info = query["template_info"]
         template_name = os.path.basename(template_info["name"])
         self.assertEqual(template_name, "flat.html")
-        self.assertEqual(template_info["context"][2]["content"].strip(), "{{ users }}")
-        self.assertEqual(template_info["context"][2]["highlight"], True)
+        self.assertEqual(template_info["context"][3]["content"].strip(), "{{ users }}")
+        self.assertEqual(template_info["context"][3]["highlight"], True)
```

Comment:
The reason for the offset is that we have another empty line between `{% extends %}` and `{% block %}`.